retsinformation.js

```diff
@@ -0,0 +1,182 @@
+		}
+		for (i = 0; i < titles.length; i++) {
+			items[titles[i].href] = clean(titles[i].text);
+		}
+		Z.selectItems(items, function(selectedItems) {
+			if (!selectedItems) return true;
+			var urls = new Array();
+			for (var i in selectedItems) {
+				urls.push(i);
+			}
+			ZU.processDocuments(urls, scrape);
+		});
+	} else {
+		scrape(doc, url);
+	};
+}
+
+function clean(text) {
+	text = text.replace(/\n/g," ");
+	text = text.replace(/ +/g," ");
+	text = text.trim();
+	return text;
+}
+
+function scrape(doc, url) {
+	var type = detectWeb(doc, url);
+	Z.debug("L91"+type);
+	var newItem = new Zotero.Item(type);
+	newItem.title = getTitle(doc, url);
+	newItem.url = url;
+	if (type=="statute") {
+		newItem.nameOfAct = newItem.title
+		newItem.codeNumber = getKortNavn(doc, url)[0];
+		newItem.publicLawNumber = newItem.codeNumber;
+		newItem.dateEnacted = getKortNavn(doc, url)[1];
+		newItem.shortTitle = getRessort(doc, url)[0];
+		if (/L(BK|OV)/.test(newItem.publicLawNumber.substr(0,3))) {
+			Zotero.debug("LOV eller LBK");
+		} else {
+			newItem = addAuthor (newItem, getRessort(doc, url)[1]);
+		};
+	} else if (type=="case") {
+		newItem.caseName = newItem.title
+		newItem = addAuthor (newItem, getRessort(doc, url)[1]);		
+		newItem.docketNumber = getKortNavn(doc, url)[0];
+		newItem.dateDecided = getKortNavn(doc, url)[1];
+		newItem.shortTitle = getRessort(doc, url)[0];
+	} else if (type=="report") {
+		newItem = addAuthor (newItem, getRessort(doc, url)[1]);
+		newItem.date = getRessort(doc, url)[2];
+		newItem.institution = "Folketinget";
+		threeLetters = getKortNavn(doc, url)[0].substr(0,3);
+		if (threeLetters=="EDP") {
+			newItem.seriesTitle = "Folketingets Ombudsmands driftundersøgelser";
+		} else if (threeLetters=="ISP") {
+			newItem.seriesTitle = "Folketingets Ombudsmands Inspektioner";
+		} else if (threeLetters=="FOU") {
+			newItem.seriesTitle = "Folketingets Ombudsmands udtalelser";
+		};
+		newItem.reportNumber = getKortNavn(doc, url)[0];
+	} else if (type=="bill") {
+		newItem = addAuthor (newItem, getRessort(doc, url)[1]);
+		newItem.billNumber = getKortNavn(doc, url)[0];
+		newItem.date = getRessort(doc, url)[2];
+	} else {
+		return;
+	};
+	newItem.complete();
+	return;
+}
+
+function getTitle (doc, url) {
+	var myXPath = '//div[@class="wrapper2"]/div/p[@class="Titel2"]';
+	title = ZU.xpathText(doc, myXPath);
+	if (!title) {
+		myXPath = '//div[@class="wrapper2"]/div/div[@id="INDHOLD"]/p[@class="Titel"]';
+		title = ZU.xpathText(doc, myXPath);
+	};
+	if (!title) {
+		myXPath = '//div[@class="wrapper2"]/div/div[@id="INDHOLD"]/font/p[@align="CENTER"]';
+		title = ZU.xpathText(doc, myXPath);
+	};
+	if (!title) {
+		myXPath = '//div[@class="wrapper2"]/div/div[@id="INDHOLD"]/h1[@class="TITLE"]';
+		title = ZU.xpathText(doc, myXPath);
+	};
+	if (!title) {
+		myXPath = '//div[@class="wrapper2"]/div/div[@id="INDHOLD"]/center/h1[@class="TITLE"]';
+		title = ZU.xpathText(doc, myXPath);
+	};
```

Comment:
you can chain all of these into ` var title = ZU.xpath(doc, ".....") || ZU.xpath(doc, ".....") || ZU.xpath(doc, ".....")`. Or, if you don't actually want to prefer one xpath over the other, you could craft a single xpath that matches all of these cases.